go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/stretchr/testify v1.6.1
 	github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2
 	go4.org v0.0.0-20190218023631-ce4c26f7be8e // indirect
-	golang.org/x/text v0.3.3 // indirect
 	gopkg.in/yaml.v2 v2.3.0
 )
+
+replace github.com/containers/common => github.com/rhatdan/common v0.10.1-0.20201009102218-77bb0d1ee370
```

Comment:
Note to self: This needs to be replaced before merging, to ensure we keep up with future fixes to c/common.

Reply:
This no longer exists.